src/neuroconv/tools/yaml_conversion_specification/yaml_conversion_specification.py

```diff
@@ -140,3 +159,40 @@ def run_conversion_from_yaml(
                     dandi_filename != ".nwb"
                 ), f"Not enough metadata available to assign name to {str(named_dandi_metadata['path'])}!"
                 named_dandi_metadata["path"].rename(str(output_folder_path / dandi_filename))
+
+    if upload_to_dandiset:
+        staging = int(upload_to_dandiset) < 100000  # temporary heuristic
+        automatic_dandi_upload(dandiset_id=upload_to_dandiset, nwb_folder_path=output_folder_path, staging=staging)
+
+
+def deploy_conversion_from_yaml_and_upload_to_dandi(  # the 'and upload to DANDI' part should ultimately be embedded in the YAML syntax
+    specification_file_path: FilePath,
+    efs_name: str,
+    dandiset_id: str,
+    transfer_method: Literal["rclone"],
```

Comment:
```suggestion
    transfer_method: Literal["rclone"] = "rclone",
```

If this is the only option then let's make it default